test/topology_custom/test_tablets_cql.py

```diff
@@ -66,3 +67,65 @@ async def alter_tablets_ks_without_waiting_to_complete():
 
     with pytest.raises(InvalidRequest, match="Can't ALTER keyspace ks, keyspace doesn't exist") as e:
         await task
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_alter_tablets_keyspace_concurrent_modification(manager: ManagerClient) -> None:
+    config = {
+        'enable_tablets': 'true'
+    }
+
+    logger.info("starting a node (the leader)")
```

Comment:
I see `the leader` part was copied from test_raft_no_quorum, but it doesn't make sense in this context (tablet test)

Reply:
Can you elaborate? What and why doesn't make sense, naming a node a leader? As far as I can tell, I need 2 nodes, so one becomes a leader and the 2nd a follower.

Reply:
I don't think we expose this nomenclature beyond raft, i.e. when we talk about scylla nodes in context different than raft (e.g. tablets) we don't designate who is the leader and who follower.

Reply:
What you're really after is not the Raft leader, but the topology coordinator. (Which happens to run on the same node.)

Reply:
I couldn't find a good name for non-coordinator nodes. For example the documentation I found was referring to them as `replicator` nodes, but it's not true in this testcase, as initially the RF=1, so the 2nd node doesn't contain any replicas, and we're not even concerned about this node having any replicas, so why resorting to this name.
So I changed it to: _starting a node (the coordinator/leader)_, whatever whoever prefers.